python/pyspark/ml/torch/distributor.py

```diff
@@ -619,8 +638,10 @@ def _run_distributed_training(
 
     @staticmethod
     def _run_training_on_pytorch_file(
```

Comment:
since `input_dataframe` is only supported when `train_object` is a function, do we need to add a check in `run`?

Reply:
The public API `TorchDistributor.run` does not have `spark_dataframe` argument.